pom.xml

```diff
@@ -110,14 +110,6 @@
           <deploy.version>1</deploy.version>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
Not sure if this code affects your tests, but my test for checking the permissions are set up correctly needs this to find the JSON API key file?